tests/test_classifier/test_confusion_matrix.py

```diff
@@ -61,3 +60,30 @@ def test_one_class(self):
         cm = ConfusionMatrix(model, classes=[0])
         cm.fit(self.X_train, self.y_train)
         cm.score(self.X_test, self.y_test)
+
+    def test_defined_mapping(self):
+        model = LogisticRegression()
+        classes = ['one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
+        mapping = {0: 'zero', 1: 'one', 2: 'two', 3: 'three', 4: 'four', 5: 'five',
+                   6: 'six', 7: 'seven', 8: 'eight', 9: 'nine'}
+        cm = ConfusionMatrix(model, classes=classes, label_encoder = mapping)
+        cm.fit(self.X_train, self.y_train)
+        cm.score(self.X_test, self.y_test)
```

Comment:
In both this test and the one below, X and y are a matrix and vector of integers, is that correct?